systemvm/debian/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -423,7 +423,7 @@ def fw_router(self):
                             "-d %s/32 -j FIREWALL_%s" % (self.address['public_ip'], self.address['public_ip'])])
             self.fw.append(["mangle", "front",
                             "-A FIREWALL_%s " % self.address['public_ip'] +
-                            "-m state --state RELATED,ESTABLISHED -j ACCEPT"])
+                            "-m state --state RELATED,ESTABLISHED -j RETURN"])
```

Comment:
it looks the code in systemvm/debian/opt/cloud/bin/configure.py need to be updated as well (search `FIREWALL_`)